lab-taylor/test/team-route-test.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
```

Comment:
These are very well done, you just need to add a 404 test for routes that are invalid in general, not just ones with improper IDs on the '/api/teams' routes.  For example, '/hello' should return a 404.